TabNine.py

```diff
@@ -53,13 +53,14 @@ def __init__(self):
         self.tabnine_proc = None
         self.num_restarts = 0
         self.install_directory = os.path.dirname(os.path.realpath(__file__))
+        self.uninstalled = False
```

Comment:
Can't we just make it so, that the restart does not use the tabnine_proc object? Seems cleaner?

Reply:
It didn't work when I tried to use a different tabnine_proc object. Have not managed to figure out why